core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -4547,151 +4504,168 @@ protected function parentView()
-							{
-								$return .= ' <a href="' . $this->addToUrl('act=' . $arrClipboard['mode'] . '&amp;mode=1&amp;pid=' . $row[$i]['id'] . '&amp;id=' . $arrClipboard['id']) . '" title="' . StringUtil::specialchars(sprintf($labelPasteAfter[1], $row[$i]['id'])) . '" onclick="Backend.getScrollOffset()">' . $imagePasteAfter . '</a>';
-							}
+						// Copy/move multiple
+						elseif ($blnMultiboard)
+						{
+							$return .= ' <a href="' . $this->addToUrl('act=' . $arrClipboard['mode'] . '&amp;mode=1&amp;pid=' . $row[$i]['id']) . '" title="' . StringUtil::specialchars(sprintf($labelPasteAfter[1], $row[$i]['id'])) . '" onclick="Backend.getScrollOffset()">' . $imagePasteAfter . '</a>';
+						}
 
-							// Drag handle
-							if (!($GLOBALS['TL_DCA'][$this->strTable]['config']['notSortable'] ?? null))
-							{
-								$return .= ' <button type="button" class="drag-handle" title="' . StringUtil::specialchars(sprintf(\is_array($labelCut) ? $labelCut[1] : $labelCut, $row[$i]['id'])) . '" aria-hidden="true">' . Image::getHtml('drag.svg') . '</button>';
-							}
+						// Paste buttons
+						elseif ($blnClipboard)
+						{
+							$return .= ' <a href="' . $this->addToUrl('act=' . $arrClipboard['mode'] . '&amp;mode=1&amp;pid=' . $row[$i]['id'] . '&amp;id=' . $arrClipboard['id']) . '" title="' . StringUtil::specialchars(sprintf($labelPasteAfter[1], $row[$i]['id'])) . '" onclick="Backend.getScrollOffset()">' . $imagePasteAfter . '</a>';
 						}
 
-						// Picker
-						if ($this->strPickerFieldType)
+						// Drag handle
+						if (!($GLOBALS['TL_DCA'][$this->strTable]['config']['notSortable'] ?? null))
 						{
-							$return .= $this->getPickerInputField($row[$i]['id']);
+							$return .= ' <button type="button" class="drag-handle" title="' . StringUtil::specialchars(sprintf(\is_array($labelCut) ? $labelCut[1] : $labelCut, $row[$i]['id'])) . '" aria-hidden="true">' . Image::getHtml('drag.svg') . '</button>';
 						}
 					}
 
-					if (\is_array($GLOBALS['TL_DCA'][$this->strTable]['list']['sorting']['child_record_callback'] ?? null))
+					// Picker
+					if ($this->strPickerFieldType)
+					{
+						$return .= $this->getPickerInputField($row[$i]['id']);
+					}
+				}
+
+				if (\is_array($GLOBALS['TL_DCA'][$this->strTable]['list']['sorting']['child_record_callback'] ?? null))
+				{
+					$strClass = $GLOBALS['TL_DCA'][$this->strTable]['list']['sorting']['child_record_callback'][0];
+					$strMethod = $GLOBALS['TL_DCA'][$this->strTable]['list']['sorting']['child_record_callback'][1];
+
+					$this->import($strClass);
+					$return .= '</div>' . $this->$strClass->$strMethod($row[$i]) . '</div>';
+				}
+				elseif (\is_callable($GLOBALS['TL_DCA'][$this->strTable]['list']['sorting']['child_record_callback'] ?? null))
+				{
+					$return .= '</div>' . $GLOBALS['TL_DCA'][$this->strTable]['list']['sorting']['child_record_callback']($row[$i]) . '</div>';
+				}
+				else
+				{
+					$label = parent::generateRecordLabel($row[$i], $this->strTable);
```

Comment:
Why do you generate the label here if it is potentially overwritten by a label_callback? Maybe we should generate it in an `else` condition instead.

Reply:
The `label_callback` requires the original label to be present.